src/algebraic_topology/simplex_category.lean

```diff
@@ -8,6 +8,7 @@ import category_theory.skeletal
 import tactic.linarith
 import data.fintype.sort
 import order.category.NonemptyFinLinOrd
+import category_theory.reflects_isomorphisms
```

Comment:
This doesn't exist any more. It might be `category_theory.functor.reflects_isomorphisms` or `category_theory.concrete.reflects_isomorphisms`

bors r-

Reply:
Thanks to all. I will put this on the queue when I get an updated `.olean` cache :-)